### PR TITLE
change startFrame to startFrameMs

### PR DESCRIPTION
**Pull request by kylebakerio**
improve clarity to address #[366](https://github.com/n5ro/aframe-extras/issues/366).

### CODE REVIEW

File: src/loaders/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ an animation and its duration:
 | repetitions       | Infinity | Number of times to play the clip, in addition to the first play. Repetitions are ignored for `loop: once`. |
 | timeScale         | 1        | Scaling factor for playback speed. A value of 0 causes the animation to pause. Negative values cause the animation to play backwards. |
 | clampWhenFinished | false        | If true, halts the animation at the last frame. |
-| startFrame        | 0        | Sets the start of an animation to a specific time (in milliseconds). This is useful when you need to jump to an exact time in an animation. The input parameter will be scaled by the mixer's timeScale. |
```
Comment (vincentfretin) [inline src/loaders/README.md:42]: The description was already correct, it's says milliseconds here.

**Review by vincentfretin — COMMENTED** [review]
I'm not in favor of merging this. Why having the unit in the name for this property and not duration or crossFadeDuration for example?
Is the Frame in the name that was confusing you? What about startAt instead of startFrame?

Also you created the PR on the wrong repo, I'm doing all changes to n5ro/aframe-extras for now.

File: src/loaders/README.md (hunk 1 above)
Comment (kylebakerio) [reply src/loaders/README.md:42]: I did see that while reviewing this. :)

At the time, I think I just saw startFrame in the schema and naturally interpreted that as one would intuitively expect--which frame to start at. It's actually startMs, in that sense, so to speak. It was also a new library that I was using while dealing with a project with very tight deadlines. Then the artist who gave me a model with animations asked that I start at X frame for animation 1, Y frame for animation 2, etc.--so, naturally, I expected startFrame to fulfill that role.

I wouldn't do 'duration', that would be a really different way to frame what it does, and we also don't indicate a 'length of time' of the part we want (though one could argue it is technically a duration of time to skip?).

I'd be perfectly fine with `startAt`. Should I make the pull request on n5ro's repo?

A bit confusing to have a c-frame branch but be managing the original; are we going to keep the c-frame version at all, now that you have management rights there?

Comment (vincentfretin) [reply src/loaders/README.md:42]: If you strongly want the renaming, you can create a PR on n5ro's repo with `startAt`, I'll merge it.
I guess I'll close the issues tracker and PR here or remove the repo completely yes. I also have the rights to release aframe-extras on npm now.

**Pull request closed without merging**